Automattic/jetpack
# WordAds: deprecate legacy widget

#### Changes proposed in this Pull Request:

﻿Now that one can add ad blocks via the block-based widget editor, let's favor that over the old legacy ad widget.

See discussion in #22016.

A few things remain to be discussed I think:

* The ad block offers more ad formats than the legacy ad widget.
* The ad block and the legacy widget have different outputs, e.g. https://github.com/Automattic/jetpack/blob/590d930bd476bef1f01aa2904a4b3d9adea58a5b/projects/plugins/jetpack/modules/wordads/class-wordads.php#L650

Is this something we would need to address, given that folks can already add ad blocks to their sidebars since WordPress 5.8?

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site with a bundle plan.
* Go to Jetpack > Settings > Traffic and enable the Ads feature.
* Go to Appearance > Widgets, and add an Ad block. 
* You should not have the option to add the legacy ad widget anymore.
